src/store.js

```diff
@@ -8,6 +8,7 @@ const store = new Vuex.Store({
   state: {
     messages: [],
     jobs: [],
+    actors: {},
```

Comment:
sorry i did no see the parsing in the API.

What i would prefer is to keep actors as an array.

then a store getter
`actorsByName: state => Object.entries(state.actors.map(a => [a.actorName, a]).`

this way you have actors to use when you want an array and use actorsByName in CMessageTable.

use https://vuex.vuejs.org/guide/getters.html#the-mapgetters-helper